src/jit/lowerxarch.cpp

```diff
@@ -4573,6 +4579,32 @@ void Lowering::SetMulOpCounts(GenTreePtr tree)
 }
 
 //------------------------------------------------------------------------------
+// SetContainsAVXFlags: default value of isFloatingType is true, we set the
+// ContainsAVX flag when floating type value is true, when SIMD vector size is
+// 32 bytes, it is 256bit AVX instruction and we set Contains256bitAVX flag too
```

Comment:
Will simplify it for something like: "Set ContainsAVX flag when it is floating type, set Contains256bitAVX flag when SIMD vector size is 32 bytes"